marketplace/google-analytics-app/src/AppConfig.tsx

```diff
@@ -0,0 +1,302 @@
+import * as React from 'react';
+import {
+  Typography,
+  Heading,
+  Paragraph,
+  TextField,
+  Option,
+  TextLink,
+  Button,
+  Select,
+  FormLabel,
+  TextInput
+} from '@contentful/forma-36-react-components';
+import styles from './styles';
+import PropTypes from 'prop-types';
+import { AppConfigParams, AppConfigState } from './typings';
+import { getSidebarLocations } from './utils';
+
+export default class AppConfig extends React.Component<AppConfigParams, AppConfigState> {
+  static propTypes = {
+    sdk: PropTypes.object.isRequired
+  };
+
+  state: AppConfigState = {
+    allContentTypes: {},
+    contentTypes: {}
```

Comment:
I would add in some defaults for the other state params.
```suggestion
    contentTypes: {},
    clientId: '',
    viewId: ''
```